src/plugins/memoryvalue/testmod_memoryvalue.c

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file
+ * @brief Tests the memory value plugin
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+
+#include <kdbconfig.h>
+#include <kdbtypes.h>
+#include <stdlib.h>
+#include <string.h>
+#include <tests_plugin.h>
+
+// the boolean parameter is needed, because we need unsigned longs as expected values, signed longs are too short thus no way to check for
+// invalid values with -1
+static void test_memoryvalue (const char * memoryvaluestring, kdb_unsigned_long_long_t memoryvalueexpected, bool errorExpected)
+{
```

Comment:
You can avoid the duplication by:

1. having a macro instead of a function (then the line numbers will be correct)
2. add the text what was tested to the assertions (which is more or less done here?)